server/models/session.js

```diff
@@ -33,30 +33,28 @@ class Session extends MongoModels {
         });
     }
 
-    static create(userId, callback) {
+    static create(userId, ip, userAgent, callback) {
 
         const self = this;
 
         Async.auto({
             keyHash: this.generateKeyHash.bind(this),
             newSession: ['keyHash', function (results, done) {
 
+                const Useragent = require('useragent');
+                userAgent = Useragent.lookup(userAgent);
```

Comment:
Would it be better to create a new variable here (`const parsedAgent = Useragent.lookup(userAgent);` maybe) instead of clobbering the argument passed in.